src/schema/persons.js

```diff
@@ -10,20 +10,52 @@ exports.typeDefs = `
     location: Address @beehiveRelation(target_type_name: "Address")
     # Current active/inactive state of the person within the network
     active: Boolean
-    roles: [SchoolAssignmentRole!] @beehiveRelation(target_type_name: "SchoolAssignmentRole", target_field_name: "person")
+    # Collection of roles Person holds at Schools
+    school_roles: [SchoolAssignmentRole!] @beehiveRelation(target_type_name: "SchoolAssignmentRole", target_field_name: "person")
+    # GSuite ID
+    gsuite_id: String
+    # GSuite profile
+    # external_email - derived from GSuite, there is a field in GSuite that records a user's personal email, this is what should be used if the user's wf email is part of the wildflowerfriends domain
+    # wf_email - derived from GSuite
+    # preferred_email - synthetic field, simple logic that uses wf_email if domain is not wildflowerfriends, otherwise uses external_email
+    # Personal Email
+    # emails: [Email]
+    # Phone Number
+    phones: [PhoneNumber]
+    # Birthday
+    birthday: Datetime  # @beehiveUTCDate
+    # Public Bio
+    public_bio: String
+    # Personal Bio
+    personal_bio: String
+    # Social Media
+    socials: [Social] @beehiveRelation(target_type_name: "Social")
+    # Websites
+    websites: [Website] @beehiveRelation(target_type_name: "Website")
+    # Montessori Credentials (AMI/AMS/other) + year + institution (string)
+    montessori_credentials: [MontessoriCredential] @beehiveRelation(target_type_name: "MontessoriCredential")
+    # Race/Ethnicity (refer to survey)
+    race_ethnicity: [RaceEthnicity] @beehiveRelation(target_type_name: "RaceEthnicity")
+    race_ethnicity_other: String
+    # Identify as POC (boolean)
+    person_of_color: Boolean
+    # Income Background 
+    income_background: IncomeBackground
+    # Language
+    language: [Language]
+    language_other: String
+    # Gender
+    gender: Gender
+    gender_other: String
+    # TL Journey
+    tl_journey: TLJourney @beehiveRelation(target_type_name: "TLJourney")
```

Comment:
will there be only one `TLJourney` per person?

Reply:
Thanks for flagging. A TLJourney will only have one person. However, one person may, although unlikely, be connected to multiple TLJourneys.